### ISSUE TITLE
qimgv stopped working for me on arch linux with kde plasma

### ISSUE THREAD

**Issue by Jieiku** [easymodo/qimgv#456]
qimgv stopped working for me on arch linux with kde plasma

when I try to launch via terminal I see an error message:

`qimgv: error while loading shared libraries: libopencv_imgproc.so.406: cannot open shared object file: No such file or directory`

**Comment by easymodo** [comment]
it needs a rebuild after opencv update

so just reinstall qimgv from AUR

**Comment by Jieiku** [comment]
I just did a full system update.

tried reinstalling qimgv from AUR, with no luck, so I uninstalled and then reinstalled, but same error.

paru qimgv

Repo (22) qt5-imageformats-5.15.8+kde+r6-1  libcdio-2.1.0-3  libcdio-paranoia-10.2+2.0.1-2  libdvdnav-6.1.1-1  glslang-11.13.0-2  spirv-tools-2022.4-1  shaderc-2022.4-1  
    libplacebo-5.229.1-2  libxpresent-1.0.1-1  mujs-1.3.2-1  hwloc-2.8.0-1  openpmix-4.2.2-1  openmpi-4.1.4-4  fftw-3.3.10-3  rubberband-3.1.2-1  uchardet-0.0.7-1  mpv-1:0.35.0-4  
    tbb-2021.5.0-2  libdc1394-2.2.6-2  cblas-3.11.0-1  libgphoto2-2.5.30-1  **opencv-4.7.0-1**
Aur (1) qimgv-1.0.2-2

qimgv: error while loading shared libraries: libopencv_imgproc.so.406: cannot open shared object file: No such file or directory

it seems opencv-4.7.0-1 was installed as a dependency but qimgv is looking for 406?
